src/wp-includes/ms-deprecated.php

```diff
@@ -646,13 +646,13 @@ function install_blog( $blog_id, $blog_title = '' ) {
 	update_option( 'home', $home );
 
 	if ( get_site_option( 'ms_files_rewriting' ) ) {
-		update_option( 'upload_path', UPLOADBLOGSDIR . "/$blog_id/files" );
+		update_option( 'upload_path', UPLOADBLOGSDIR . "/$blog_id/files", false );
```

Comment:
This should remain autoloaded as it may be regularly accessed in multisite.
```suggestion
		update_option( 'upload_path', UPLOADBLOGSDIR . "/$blog_id/files" );
```